src/librustc_mir/hair/pattern/check_match.rs

```diff
@@ -238,8 +238,8 @@ impl<'a, 'tcx> MatchVisitor<'a, 'tcx> {
                                                         is non-empty",
                                                        pat_ty));
                     span_help!(&mut err, scrut.span,
-                               "Please ensure that all possible cases are being handled; \
-                                possibly adding wildcards or more match arms.");
+                               "ensure that all possible cases are being handled; \
```

Comment:
```suggestion
                               "ensure that all possible cases are being handled, \
```

Actually, while we're here, could we also change this semicolon to a comma? The "possibly by adding" isn't an [independent clause](https://writing.wisc.edu/Handbook/Semicolons.html).